src/platform/Zephyr/ConfigurationManagerImpl.cpp

```diff
@@ -54,6 +54,10 @@
 #include <platform/ThreadStackManager.h>
 #endif
 
+#if CHIP_DEVICE_LAYER_TARGET_NRFCONNECT
+#include <mpsl/mpsl_lib.h>
```

Comment:
is the mpsl is a common files for all platforms based on zephyr?
or it should be defined in nrfconnect platform?

Reply:
+1, mpsl is nordic specific, so ideally we should not do that for Zephyr, otherwise there may be problems with contribution to upstream.

Reply:
It is Nordic-specific hence `CHIP_DEVICE_LAYER_TARGET_NRFCONNECT`. Otherwise, we need to implement the whole factory reset method in the nrfconnect platform, because currently, we use the Zephyr one from here. Do you think it is better to move it there?